HashRegistrarSimplified.sol

```diff
@@ -451,7 +451,8 @@ contract Registrar {
         if(address(h.deed) != 0) {
             // Reward the discoverer with 50% of the deed
             // The previous owner gets 50%
-            h.deed.setBalance(h.deed.value()/2);
+            h.value = max(h.value, minPrice);
```

Comment:
`minPrice` is greater than `h.value` if only one bid was received - eg, in the case of an uncontested auction. 

The deed's balance will always be greater than or equal to `h.value`, because when a winning bid is revealed, the `actualValue` is set to the minimum of the deed's balance and the declared bid. `h.value` is then set to `actualValue`.